neuralpp/test/quick_tests/symbolic/normalizer_test.py

```diff
@@ -115,27 +115,36 @@ def test_quantifier_normalizer():
     j = BasicVariable('j', int)
     sum_ = BasicSummation(int, i, empty_context & (j < i) & (i < 10), i + j)
     context = empty_context & (j == 5)
-    assert normalizer.normalize(sum_, context).syntactic_eq(
-        BasicSummation(int, i, empty_context & (j < i) & (i < 10), 5 + i))  # SymPy switched 5 and i
+    # 6 + 7 + 8 + 9 + 5 * 4 = 50
+    assert normalizer.normalize(sum_, context).syntactic_eq(BasicConstant(50))  # SymPy switched 5 and i
 
     context = empty_context & (j == 10)
     assert normalizer.normalize(sum_, context).syntactic_eq(BasicConstant(0, int))
 
-    sum_ = BasicSummation(int, i, empty_context & (j < i), if_then_else(j > 5, i + j, i))
+    k = BasicVariable('k', int)
+    jj, kk = sympy.symbols('j k')
+    sum_ = BasicSummation(int, i, empty_context & (j < i) & (i < k), if_then_else(j > 5, i + j, i))
```

Comment:
nit: using `empty_context & ` to create contexts from expressions seems a little unnatural. Why not create something like `Context.from_expression(expr)`?

Reply:
I've changed to that.